core/object/make_virtuals.py

```diff
@@ -47,8 +47,8 @@
 }\\
 _FORCE_INLINE_ bool _gdvirtual_##m_name##_overridden() const { \\
 	ScriptInstance *_script_instance = ((Object*)(this))->get_script_instance();\\
-	if (_script_instance) {\\
-	    return _script_instance->has_method(_gdvirtual_##m_name##_sn);\\
+	if (_script_instance && _script_instance->has_method(_gdvirtual_##m_name##_sn)) {\\
+        return true;\\
```

Comment:
```suggestion
		return true;\\
```

Reply:
I ...think I changed it to tabs? Looks like it was previously spaces...

Reply:
It was tabs and spaces and you changed to spaces only

Reply:
In the latest push, it looks likes it's just tabs now on the lines that @anrp changed.

Probably the rest of the tabs/spaces issues should be addressed in a separate PR?

Reply:
Do you want it to be a tab and then spaces (like it was) or just tabs (requested?)

Reply:
Just tabs, as requested :)

Reply:
FYI AFAICT these two lines were changed to tabs as of 4c33c60 (I did notice that the email I saw changed to spaces, but looking at the PR on disk/copy/pasting from web UI shows tabs)